docs/release-notes/16-6-0/README.md

```diff
@@ -3,24 +3,103 @@ title: OpenProject 16.6.0
 sidebar_navigation:
     title: 16.6.0
 release_version: 16.6.0
-release_date: 2025-10-23
+release_date: 2025-11-05
 ---
 
 # OpenProject 16.6.0
 
-Release date: 2025-10-23
+Release date: 2025-11-05
 
-We released OpenProject [OpenProject 16.6.0](https://community.openproject.org/versions/1413).
-The release contains several bug fixes and we recommend updating to the newest version.
-In these Release Notes, we will give an overview of important feature changes. At the end, you will find a complete list of all changes and bug fixes.
+We released [OpenProject 16.6.0](https://community.openproject.org/versions/1413). The release contains several bug fixes and we recommend updating to the newest version. In these Release Notes, we will give an overview of important feature changes. At the end, you will find a complete list of all changes and bug fixes.
 
 ## Important feature changes
 
-<!-- Inform about the major features in this section -->
+Take a look at our release video showing the most important features introduced in OpenProject 16.6.0:
 
-## Important updates and breaking changes
+![Release video of OpenProject 16.6](https://openproject-docs.s3.eu-central-1.amazonaws.com/videos/OpenProject_16_6_release.mp4)
 
-<!-- Remove this section if empty, add to it in pull requests linking to tickets and provide information -->
+### Calculated values for project evaluation and scoring (Enterprise add-on)
+
+OpenProject 16.6 introduces new project attribute types that make portfolio evaluation measurable and transparent: Hierarchy, Scored list, and Calculated value:
+
+![OpenProject administration with new types of project attributes: Calculated value, Hierarchy and Scored list are highlighted.](openproject-16-6-project-attributes.png)
+
+**Hierarchy** project attributes let you organize project information in a structured, nested way — for example, grouping initiatives by region, department, or program.
+
+![OpenProject administration shows a project attribute type hierarchy: The name is 'Strategic initiative' and it shows 3 items with sub-items, e.g. 'Digital transformation', and the options to edit, add sub-items, change parent and more.](openproject-16-6-project-attributes-hierarchy.png)
+
+**Scored lists** extend this concept by allowing administrators to assign numeric scores to list options, such as effort levels or risk ratings. These values can then be used as input for calculated results.
```

Comment:
Please be aware, that the name for 'Scored list' might still change. At least that is what @dominic-braeunlein mentioned to me today.